File: Day01/day01.py
f = open("input.txt", "r")
lines = f.readlines()
elves = []
current = 0

for i in range(len(lines)):
    x = lines[i]
    if (x == "\n"):
        elves.append(current)
        current = 0 
        continue

    current += int(x)
elves.append(current)

print(max(elves))

elves.sort()
print(sum(elves[-3:]))

File: Day01/Day01.r
input <- readLines("input.txt")
input <- as.integer(input)

elves <- c()
calories <- 0

for(x in input) {
  if (is.na(x)) {
    elves <- append(elves, calories)
    calories <- 0
    next
  }
  calories <- (calories + x)
}
elves <- append(elves, calories) # loop will not catch final block

print(max(unlist(elves)))

elves <- sort(elves)
print(sum(tail(elves, 3)))
